index.js

```diff
@@ -156,22 +183,34 @@ function getOptionsArgument(serviceOrOptions?: string | Options) {
     : serviceOrOptions;
 }
 
+function getMinimumSecurityLevel(minimumSecurityLevel?: SecMinimumLevel) {
+    if (minimumSecurityLevel === undefined) {
```

Comment:
this function can be simplified to `return minimumSecurityLevel || SECURITY_LEVEL.ANY;`

Reply:
it is different now, but I tried to keep it simple